util/db/migrations/20210601161145-delete-registration.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
```

Comment:
Could you rename this file to better describe the action the migration takes? The filename is saved in the Sequelize Meta table, so a good descriptive name is essential.

It doesn't 'delete a registration' but really 'creates the revocation table', so `util/db/migrations/20210601161145-create-revocation.js` would be a better name for it.

Reply:
resolved in https://github.com/Scottish-Natural-Heritage/trap-registration-api/pull/25/commits/ec4ffc241448f099d9575dbc2e68a8bd207a7d8b